plugins/extractors/mariadb/README.md

```diff
@@ -0,0 +1,45 @@
+# mariadb
+
+## Usage
+
+```yaml
+source:
+  type: mariadb
+  config:
+    user_id: admin
```

Comment:
@scortier should we use connection url for this instead?

Reply:
sure, will change it